doc/ale.txt

```diff
@@ -2208,6 +2208,17 @@ ALE will use to search for Python executables.
 ===============================================================================
 8. Commands/Keybinds                                             *ale-commands*
 
+ALEComplete                                                       *ALEComplete*
+
+  Manually trigger LSP autocomplete and show the menu. Works only when called
+  from insert mode. >
+
+    inoremap <silent> <C-Space> <C-\><C-O>:GetCompletions<CR>
```

Comment:
```suggestion
    inoremap <silent> <C-Space> <C-\><C-O>:ALEComplete<CR>
```

I think you need to update the command here.

Reply:
Thanks, that should be fixed now!